.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,15 +2,44 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: ''
+labels: 'bug'
 assignees: ''
 
 ---
 
 **Environment**
 - Loader: [Forge/Fabric]
+- Loader Version:[]  <!-- Do not put 'latest' as a  version that is not valid  -->
 - Mod version: [e.g. 1.18.2-12.0]
 - OS: [e.g. MacOS, Windows]
 
 **Describe the bug**
 A clear and concise description of what the bug is.
+
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+<!-- in the [] put an X inside of the box eg. [X] -->
+
+- The issue occurs in...
+   - [ ] Singleplayer
+   - [ ] Multiplayer
```

Comment:
Multiplayer and single player a are different and can cause bugs completely differently